targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LINKER_FILE device/TOOLCHAIN_ARM_STD/MBRZA1LU.sct)
+    set(STARTUP_FILE device/TOOLCHAIN_ARM_STD/startup_RZ_A1LU.S)
+    target_sources(mbed-core
+        INTERFACE
+            device/TOOLCHAIN_ARM_STD/weak_handler.S
+    )
+elseif(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(LINKER_FILE device/TOOLCHAIN_GCC_ARM/RZA1LU.ld)
+    set(STARTUP_FILE device/TOOLCHAIN_GCC_ARM/startup_RZ1ALU.S)
+    target_sources(mbed-core
+        INTERFACE
+            device/TOOLCHAIN_GCC_ARM/weak_handler.S
+    )
+elseif(${MBED_TOOLCHAIN} STREQUAL "IAR")
+    set(LINKER_FILE device/TOOLCHAIN_IAR/MBRZA1LU.icf)
+    set(STARTUP_FILE device/TOOLCHAIN_IAR/startup_RZA1LU.S)
+    target_sources(mbed-core
+        INTERFACE
+            device/TOOLCHAIN_IAR/weak_handler.S
+    )
+endif()
+
+set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+
+target_sources(mbed-core
+    INTERFACE
+        device/RZ_A1_Init.c
+        device/cmsis_nvic.c
+        device/mbed_sf_boot.c
+        device/mmu_RZ_A1LU.c
+        device/nvic_wrapper.c
+        device/os_tick_ostm.c
+        device/system_RZ_A1LU.c
+        trng_api_esp32.cpp
+        PeripheralPins.c
+
+        ${STARTUP_FILE}
+)
+
+target_include_directories(mbed-core
+    INTERFACE
+        .
+        device
+        device/inc
+        device/inc/iobitmasks
+        device/inc/iodefines
+        TARGET_MBED_MBRZA1LU
```

Comment:
This should not be there. It should only be listed as an include directory if `MBED_MBRZA1LU` is a listed label.
```suggestion
```